src/index.html

```diff
@@ -135,13 +135,14 @@ <h4>
                 </h4>
                 <section class="presentation">
                         <p>
-                            Guangzhou Fionaa Company Limited., main activity scope is import and export business which involve sourcing, purchasing, quality control and International-Shipping.
-                            The company can do both wholesale and retail trade;
+                            Guangzhou Fionaa Company Limited., activity scope is import and export business trade which involve:
+                            Gathering Information; Factories Reasearch; Negotiation & Quoting; Sampling; Purchasing; Quality Control; 
+                            Shipping and Loacl delivery.The company can do both wholesale and retail trade;
                         </p> 
 
                         <p>
                             With more than 13years of experience our company team has a certain scale and strength.
-                            We mainly export a huge variety of goods such as machinery, building materials, lighting, furniture, home appliance ...
+                            We mainly export a huge variety of goods such as machinery, building materials, lighting, furniture, home appliances ...

```

Comment:
home appliances and decorations could be better

Reply:
Here also decorations was added.